internal/executor/helm/executor.go

```diff
@@ -131,6 +132,16 @@ func (e *Executor) Execute(ctx context.Context, in executor.ExecuteInput) (execu
 	}
 }
 
+func (Executor) Help(ctx context.Context) (interactive.Message, error) {
+	return interactive.Message{
+		Base: interactive.Base{
+			Body: interactive.Body{
+				CodeBlock: (&HelpCommand{}).Help(),
```

Comment:
The `HelpCommand` struct is no longer needed if you'll get rid of the unused logic. So then you can simplify it - either keep `Help() string` as a simple function, or embed it directly here in the codeblock - both option work for me 👍